docs/concepts/fs/feature_group/feature_pipelines.md

```diff
@@ -32,10 +32,14 @@ Spark is popular as a feature engineering framework as it can scale to process l
 
 SQL has grown in popularity for performing heavy lifting in feature pipelines - computing aggregates on data - when the input data already resides in a data warehouse. Data warehouses also support data validation, for example, through Great Expectations in DBT. However, SQL is not mature as a platform for transformations and dimensionality reductions, where UDFs are applied row-wise.
 
-
 You can do aggregation in SQL for data in your data warehouse or database.
 
+
 ### Feature Engineering in Flink
+Apache Flink is a powerful and flexible framework for stateful feature computation operations over unbounded and bounded data streams. It is used for feature engineering when you need very fresh features computed in real-time. Flink provides a rich set of operators and functions such as time windows and aggregation operations that can be applied to keyed and/or global window streams. Flink’s stateful operations allows it to maintain and update state across multiple data records or events, which is particularly useful for feature engineering tasks such as sessionization and/or maintaining rolling aggregates over a sliding window of data.
+
+Currently, Flink feature engineering pipelines are supported in Java/Scala only. Hopsworks Feature Store expects that your computed features are encapsulated in POJO that has the same schema as the feature group you are writing into. In database terms this POJO class corresponds to one row. Feature Group is expected to be online and stream enabled.
```

Comment:
This part belongs to the user guides. The concepts are more high level description.

Reply:
Also the feature group only needs to be `stream` enabled - it doesn't have to be online enabled.